doc/types/transform-options.md

```diff
@@ -33,6 +33,9 @@ name on the global object in exports.
 
 The other possible format values are "steal","amd", and "cjs".
 
+@option {Boolean} [noGlobalShim=false] By default, if _options.format_ is `"global"` then a shim will be added
+for the global _define_ method. Setting _noGlobalShim_ to `true` prevents this.
+
```

Comment:
This is good, perhaps also explain that the global shim is so that the script can be ran standalone, and exluding the shim means it will have to be ran with an AMD loader like requirejs.